embeddings.go

```diff
@@ -103,7 +103,7 @@ var stringToEnum = map[string]EmbeddingModel{
 // then their vector representations should also be similar.
 type Embedding struct {
 	Object    string    `json:"object"`
-	Embedding []float64 `json:"embedding"`
+	Embedding []float32 `json:"embedding"`
```

Comment:
We are trying to track down a bug, and I think this change causes problems when you use Python to store embeddings and golang to try to do similarity scores against them. Python seems to be using float64 (or something with even more precision), and when we try to use those values with the embeddings from this library we get wildly different results then when we do the same code using Python only. I am not sure if the endpoint returns 64, but it would seem down casting to 32 bit make more sense then losing precision you can't get back.